### DISCUSSION TITLE
Private templates organisation

### DISCUSSION THREAD

**Discussion by EmilienDreyfus** [langchain-ai/langsmith-sdk#581] [Ideas]
Hi there,

In my company, we store our private prompt templates on Langsmith to benefit from versioning, datasets, and testing decoupled from our codebase. We also use it in production by pinning a commit and using cache on our side. It's really cool!

However, we are facing an issue with the growth of our prompt usage, and it would be very useful for us to be able to organize our prompts better, with tags and directories. We're not interested in Prompt Hub, and I think that private prompt management is very underrated in the interface.

What do you think about that?

**Comment by madams0013** [comment]
Hi! Thanks for the message! This is great feedback.

We're currently in the process of working on making private prompts separate from the public "hub". Organization of private prompts will also be a focus on that front. 

I'll update in this thread as related features come out.

**Comment by hinthornw** [comment]
Hi @EmilienDreyfus we've launched the private prompts feature - how is it serving your needs?
